### DISCUSSION TITLE
Loading (N/m) calculation of Propeller

### DISCUSSION THREAD

**Discussion by ilm71** [byuflowlab/FLOWUnsteady#127] [Q&A] [answered]
Dear All, 

currently I am comparing my BEMT results to FlowUnsteady by comparing the Propeller loading (N/m) 

<img width="354" alt="loading" src="https://github.com/byuflowlab/FLOWUnsteady/assets/137082953/f472d419-7f86-4b94-acbf-2e165690acb2">

How was the loading of the validation case calculated? I have implemented in the "run_simulation" the "debug=true" line to gain any information out of the .vtk files. 

Thank you very much for your answer! 

Best

**Comment by EdoAlvarezR** [comment, marked as answer]
Hi Philipp!

The "Rotor Hover" tutorial has an example on how to read the blade loading from the VTK files and take a time average:

* [Tutorial](https://flow.byu.edu/FLOWUnsteady/examples/rotorhover-aero/)
* [Code](https://github.com/byuflowlab/FLOWUnsteady/blob/master/examples/rotorhover/rotorhover_postprocessing.jl)

**Comment by ilm71** [comment]
Hi Ed, 

many thanks for your quick reply. So as far I can understand, the blade loading is an average loading of the fluctuations on the blade caused by the rotations? 

My first intention was, that the blade loading is the "Thrust[i]/chord[i] "distribution

**Comment by ilm71** [comment]
And compared to that: What is meant by the  "Distributed_Load_Magnitude"? How can that be expressed?

![image](https://github.com/byuflowlab/FLOWUnsteady/assets/137082953/5fdc8ef6-71a6-4f88-af51-138f8775c0d2)

**Comment by EdoAlvarezR** [comment]
Blade loading (or distributed load) is force per unit span, hence it has unit of N/m. The simulation computes the instantaneous loading in each time step, which depending on your case it might fluctuate in time (due to blade vortex interaction, edgewise flow, etc), hence the need of time averaging in.

**Comment by EdoAlvarezR** [comment]
Be aware that the `DistributedLoad` vector along the blade contains both the normal and tangential force. The rotor thrust comes from integrating the normal force over each blade, while rotor torque is computed from integrating position x tangential force over each blade. In the literature (and in the figure that you are asking about), blade loading is the distributed **normal** force.

**Comment by ilm71** [comment]
Okay thanks for the input - I was just wondering that if I devide the DistributedLoad by 1m(1000mm), I have for a hover propeller the same distribution as for the C_t distribution from my BEMT.

**Comment by ilm71** [comment]
and therefore I am wondering if this is reasonable - what do you think?

**Comment by EdoAlvarezR** [comment]
The distributed load is the force already divided by the span length, so it will end up with the wrong units. It also depends on how you are defining C_t in your BEMT.

**Comment by ilm71** [comment]
Within the BEMT, C_t is defined based on the sectional Thrust as a function of (phi) -->  (Lift*cos(phi)-Drag*sin(phi)), where phi is the angle between the free-stream velocity and the propeller plane

**Comment by ilm71** [comment]
But when des DistributedLoad is the Normal-Force (I guess normal to the propeller plane), the Loading should be equivalent to the thrust distribution in hover in FLOWUnsteady - am I right?

**Comment by EdoAlvarezR** [comment]
> Be aware that the DistributedLoad vector along the blade contains both the normal and tangential force.
You'll have to decompose DistributedLoad into the direction that makes the normal force to get the thrust distribution.

From the normal force distribution you can then compute C_t in a manner that is consistent with your BEMT outputs (probably `C_t = N / (0.5 * rho * Uinf^2 * chord)` where `N` is the normal force at that station, perhaps multiplied by the number of blades depending on how your BEMT defined its outputs).

Alternatively, you can follow [this discussion](https://github.com/byuflowlab/FLOWUnsteady/discussions/96) to output the tangential (`ct`) and normal (`cn`) force coefficients that were calculated internally, which probably resemble the definition of your BEMT (see [this](https://github.com/byuflowlab/FLOWVLM/blob/master/src/FLOWVLM_rotor.jl#L2083C1-L2087C49))

**Comment by ilm71** [comment]
Thanks for your quick feedback - I have already plotted the internaly C_t values at a specific time step and compared this value with my C_t BEMT Results at the predefined blade sections - the results are not compareable, but the overall thrust and torque are matching exactly. 

That's why I am wondering where the error could be

**Comment by EdoAlvarezR** [comment]
'ct' in FLOWUnsteady is the tangential force coefficient, while 'cn' is the normal force coefficient. My guess is that C_t in your BEMT is a "thrust" coefficient, so you actually want to compare C_t against 'cn'. Also, it's possible that your BEMT already multiplied the coefficient by the number of blades.

**Comment by ilm71** [comment]
Thanks Ed for the answer - I have already plotted the results previously and the "c_n" value gained within FlowUnsteady is slightely higher compared to my BEMT results -nevertheless, the plot looks compareable ... the overall Thrust and Torque are matching quite well too, now I have to figure out why FlowUnsteady predicts higher C_n values than the BEMT

**Comment by EdoAlvarezR** [comment]
The main difference is that FLOWUnsteady is a wake-resolving simulation capturing contraction of the streamtube and all other wake dynamics (these become particularly important in near hover conditions). BEMT on the other hand simply assumes a straight semi-infinite cylindrical streamtube (which is baked into the momentum balance part of the theory that leads to the far field induced velocities being double the near field induced velocities).

With that in mind, you should expect FLOWUnsteady to be more accurate than BEMT.

**Comment by EdoAlvarezR** [comment]
See this VPM vs experimental vs URANS vs BEMT comparison in the tutorials for a rotor in hover: [link](https://flow.byu.edu/FLOWUnsteady/examples/rotorhover-aero)

(BEMT gets more accurate as you ramp up the advance ratio)

**Comment by ilm71** [comment]
Yes, that‘s why I am
thinking about the possible error - I am not using the BEMT implemented within the Flowunsteady code but using my own BEMT model, therefore differenced may occur…

**Comment by ilm71** [comment]
> > Be aware that the DistributedLoad vector along the blade contains both the normal and tangential force.
> > You'll have to decompose DistributedLoad into the direction that makes the normal force to get the thrust distribution.
> 
> From the normal force distribution you can then compute C_t in a manner that is consistent with your BEMT outputs (probably `C_t = N / (0.5 * rho * Uinf^2 * chord)` where `N` is the normal force at that station, perhaps multiplied by the number of blades depending on how your BEMT defined its outputs).
> 
> Alternatively, you can follow [this discussion](https://github.com/byuflowlab/FLOWUnsteady/discussions/96) to output the tangential (`ct`) and normal (`cn`) force coefficients that were calculated internally, which probably resemble the definition of your BEMT (see [this](https://github.com/byuflowlab/FLOWVLM/blob/master/src/FLOWVLM_rotor.jl#L2083C1-L2087C49))

I guess for the chord you used the mean-chordlength?

**Reply by EdoAlvarezR** [reply]
Negative. The chord used for the local coefficients along the blade (`cl`, `cd`, `cn`, and `ct`) are all based on local quantities. Meaning, the chord and velocity used in the denominator of the coefficient are the local chord length and local effective velocity (i.e., as opposed to just `Uinf`).

**Reply by ilm71** [reply]
thanks, I have already figured out how the quantities were calculated within your code in the meanwhile - After some adoptions of my code I am able to compare the blade loading to the FLOWUnsteady results now - many thanks for your support!

**Reply by EdoAlvarezR** [reply]
I'm glad to hear that!
